emoji/src/main/java/com/vanniktech/emoji/RecentEmojiGridView.java

```diff
@@ -18,7 +18,7 @@ public RecentEmojiGridView init(@Nullable final OnEmojiClickedListener onEmojiCl
     recentEmojis = recentEmoji;
 
     final Collection<Emoji> emojis = recentEmojis.getRecentEmojis();
-    emojiArrayAdapter = new EmojiArrayAdapter(getContext(), emojis.toArray(new Emoji[emojis.size()]), onEmojiClickedListener);
+    emojiArrayAdapter = new EmojiArrayAdapter(getContext(), emojis.toArray(new Emoji[emojis.size()]), onEmojiClickedListener, null);
```

Comment:
We should also here add the ability to long press and select a different skin tone